vercel/data_source_project.go

```diff
@@ -220,6 +220,21 @@ For more detailed information, please see the [Vercel documentation](https://ver
 					},
 				},
 			},
+			"options_allowlist": schema.SingleNestedAttribute{
+				Description: "Ensures only requests starting with specified paths can bypass Deployment Protection for OPTIONS requests.",
+				Computed:    true,
+				Attributes: map[string]schema.Attribute{
+					"paths": schema.ListAttribute{
+						Description: "The path to compare the request path against.",
+						Computed:    true,
+						ElementType: types.ObjectType{
+							AttrTypes: map[string]attr.Type{
+								"value": types.StringType,
+							},
+						},
```

Comment:
How come it's a list ob objects, not just a list of strings?

Reply:
Making it flexible for the future, leaves room for configurations of the paths. For example, Trusted IPs has notes. For this feature I can see supporting regex, or requiring specific headers on the request etc.

Reply:
Gotcha. Well, this LGTM 🙏

Reply:
To make sure I fully answered your question, this structure is a 1-1 reflection of the db / api structure